docs/publishers/events.rst

```diff
@@ -0,0 +1,46 @@
+Events
+======
+
+Hypothesis client emits a few custom DOM events at ``document.body``, to which your app can react.
+
+.. option:: hypothesis:layoutchange
+
+   This event is emitted when the sidebar layout changes. eg. when it is opened or closed.
+
+   It extends `CustomEvent <https://developer.mozilla.org/docs/Web/API/CustomEvent>`_, so its payload can be found inside the ``detail`` property.
```

Comment:
This applies to all events, not just this event, so it should either be part of the introduction or a separate paragraph after the list of events.